components/Common/ShellBox/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRef } from 'react';
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import type { FC, PropsWithChildren, MouseEvent, ReactNode } from 'react';
+
+type ShellBoxProps = {
+  children: string | ReactNode;
+  textToCopy?: string;
+};
+
+const ShellBox: FC<ShellBoxProps> = ({
```

Comment:
This should be:

```tsx
const ShellBox: FC<PropsWithChildren<ShellBoxProps>> = ({
  children,
  textTopCopy,
}) => { /* code of the component */ }
```

Reply:
The Prop type you're setting on `FC` mismatches the one after the object. 
Also when you use `FC` you don't need to set the types of the props object :)